src/Transport/Topology/IConventions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace NServiceBus.AzureServiceBus
+{
+    using System;
+
+    interface IConventions
+    {
+        bool IsMessageType(Type type);
+    }
+
+    class ConventionsAdapter : IConventions
```

Comment:
Navigating to `ConventionsAdapter` in GH won't be possible if it's sharing a file with another type.